io.catenax.days_of_supply/1.0.0/DaysOfSupply.ttl

```diff
@@ -0,0 +1,144 @@
+   samm:description "Set of allocated Days of Supply that may relate to different stock locations."@en ;
+   samm:dataType :AllocatedDaysOfSupply .
+
+:AllocatedDaysOfSupply a samm:Entity ;
+   samm:preferredName "Days Of Supply Allocated to a Partner"@en ;
+   samm:description """This defines the number of days with allocated supply for an item stock in a given location that is available for the use in production or deliveries. 
+The allocated days of supply are not available for other customers."""@en ;
+   samm:properties ( :amountOfAllocatedDaysOfSupply :stockLocationBPNS :stockLocationBPNA ) .
+
+:amountOfAllocatedDaysOfSupply a samm:Property ;
+   samm:preferredName "Amount of Allocated Days Of Supply"@en ;
+   samm:description "Amount of Days of supply of a given allocated item stock in a given location."@en ;
+   samm:characteristic :QuantityCharacteristic .
+
+:stockLocationBPNS a samm:Property ;
+   samm:preferredName "BPNS of Stock Location"@en ;
+   samm:description "BPNS identifying the site the stock is located at. One stock has always one location."@en ;
+   samm:see <https://catena-x.net/fileadmin/user_upload/Standard-Bibliothek/Update_PDF_Maerz/5_BPDM/CX_-_0010_BUSINESS_PARTNER_NUMBER_PlatformCapabilityBPDM_v_1.0.1.pdf> ;
+   samm:characteristic ext-number:BpnsTrait ;
+   samm:exampleValue "BPNS1234567890ZZ" .
+
+:stockLocationBPNA a samm:Property ;
+   samm:preferredName "BPNA of Stock Location"@en ;
+   samm:description "BPNA identifying the address the stock is located at. One stock has always one location."@en ;
+   samm:see <https://catena-x.net/fileadmin/user_upload/Standard-Bibliothek/Update_PDF_Maerz/5_BPDM/CX_-_0010_BUSINESS_PARTNER_NUMBER_PlatformCapabilityBPDM_v_1.0.1.pdf> ;
+   samm:characteristic ext-number:BpnaTrait ;
+   samm:exampleValue "BPNA1234567890ZZ" .
+
+:QuantityCharacteristic a samm-c:SortedSet ;
+   samm:preferredName "Quantity Characteristic"@en ;
+   samm:description "List of quantity of days of supply. It MUST include at least current date and it is recommended to include future dates."@en ;
+   samm:dataType :QuantityOfDaysOfSupply .
+
+:QuantityOfDaysOfSupply a samm:Entity ;
+   samm:preferredName "Quantity of Days of Supply"@en ;
+   samm:description "Days of supply for a specific date."@en ;
+   samm:properties ( :date :daysOfSupply ) .
+
+:date a samm:Property ;
+   samm:preferredName "Date of Days of Supply"@en ;
+   samm:description "Date refers to the specific calendar day (current or projected) associated with the measured (or expected) inventory level. It serves as a timestamp for calculating Days of Supply, indicating when the inventory count was taken or projected."@en ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2024-01-01T14:23:00+01:00"^^xsd:dateTime .
+
+:daysOfSupply a samm:Property ;
+   samm:preferredName "Days of Supply"@en ;
+   samm:description """Amount of days, before the current stock is expected to be exhausted. 
```

Comment:
please correct the spacing

Reply:
Hi, thanks for the feedback. Can you elaborate if this is referring to the spacing in the formulas/description of Days of Supply property? Thanks for the review

Reply:
We removed the double break line from the description. Did you mean that?